tests/src/Kernel/fixtures/markup_rendering_patterns/timeline.yml

```diff
@@ -0,0 +1,168 @@
+timeline_items_hide:
+  render:
+    '#type': pattern
+    '#id': timeline
+    '#fields':
+      settings:
+        hide_from: 5
+        hide_to: -2
+      toggle_collapsed: "Show %d more items"
+      toggle_expanded: "Hide %d items"
+      items:
+        - id: "1"
+          label: "13 September 2017"
+          title: "Item title"
+          content:
+            '#markup': <a href="/example">President Juncker's State of the Union speech</a>
+        - id: "2"
+          label: "28-29 September 2017"
+          content:
+            '#markup': <a href="/example">Informal Digital Summit, Tallinn</a>
+        - id: "3"
+          label: "14 November 2017"
+          content: "Strengthening European identity through education and culture: European Commission's contribution to the Leaders' meeting."
+        - id: "4"
+          label: "17 November 2017"
```

Comment:
could some fields have only label, some only content, some only title ? PS: I dont see any item that uses title